modules/microshift-downloading-container-images.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * microshift/running_applications/microshift-deploy-with-mirror-registry.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-downloading-container-images_{context}"]
+= Downloading container images
+
+After you have located the container list and completed the mirroring prerequisites, download the container images to a host with Internet access.
+
+.Prerequisites
+
+* You are logged into a host with access to `microshift-quay`
+* You have ensured that the ``.pull-secret-mirror.json` file and `microshift-containers` directory contents are available locally.
```

Comment:
The document I am reading suggests that you edit .pull-secret.json not create a file called .pull-secret-mirror.json

Am I following the wrong document ?
https://docs.openshift.com/container-platform/4.13/installing/disconnected_install/installing-mirroring-installation-images.html#installation-adding-registry-pull-secret_installing-mirroring-installation-images

Reply:
Steps 1-3 of that procedure seem to be the creation part. @ggiguash Could you verify that this is the link we need here?

Reply:
Step 2 in the procedure is creating a new file, which is then edited by appending the mirror-specific info.